src/tuzue/ui/curses.py

```diff
@@ -202,6 +202,13 @@ def interact(self, view):
         return self.input_process(view, key, keyname)
 
     def input_process(self, view, key, keyname):
+        # this is for mac osx  Mac osx 10.15.7 TODO: Test on other platforms.
+        if key == 127:
+            keyname = b"KEY_BACKSPACE"
+        if keyname in self.edit_actions:
+            self.edit_actions[keyname](view)
```

Comment:
Thank you for looking into this

What happens if this "if" is not here? We do something very similar further down

Reply:
Can you please elaborate further on your comment?


I will test this on ubuntu 22.04 LTS on arm64 to see if there are any issues. 

Unlike go, rust and c. Python allows us to ignore any errors should there be one so we can keep working. 

Are you open to using ``try`` and ``except`` block to catch any worries, then ``finally``  just continue.